src/engines/html5/html5.js

```diff
@@ -941,4 +941,50 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     }
     return extended;
   }
+
+  enterPip(): void {
```

Comment:
Change to ```enterPictureInPicture```
Pip is not a known acronyms, but if you already use it so use capital letters -> PIP

Reply:
changed